<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsAppearance</class>
 <widget class="QWidget" name="SettingsAppearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="outputGroup">
     <property name="title">
      <string>Output Pane</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="OutputFont_label">
        <property name="text">
         <string>Font Family</string>
        </property>
        <property name="buddy">
         <cstring>OutputFont</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="OutputFont"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="OutputFontSize_label">
        <property name="text">
         <string>Font Size</string>
        </property>
        <property name="buddy">
         <cstring>OutputFontSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="OutputFontSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="inputGroup">
     <property name="title">
      <string>Input Pane</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="InputFont_label">
        <property name="text">
         <string>Font Family</string>
        </property>
        <property name="buddy">
         <cstring>InputFont</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="InputFont"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="InputFontSize_label">
        <property name="text">
         <string>Font Size</string>
        </property>
        <property name="buddy">
         <cstring>InputFontSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="InputFontSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="InputForeground_label">
        <property name="text">
         <string>Foreground</string>
        </property>
        <property name="buddy">
         <cstring>InputForeground</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ColorPickerButton" name="InputForeground">
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="InputBackground_label">
        <property name="text">
         <string>Background</string>
        </property>
        <property name="buddy">
         <cstring>InputBackground</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ColorPickerButton" name="InputBackground">
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4000</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
